fleece/httperror.py

```diff
@@ -11,6 +18,13 @@ class HTTPError(Exception):
     def __init__(self, status=None, message=None):
         """Initialize class."""
         responses = BaseHTTPRequestHandler.responses
+
+        # Add some additional responses that aren't included...
+        responses[418] = ('I\'m a teapot', SECRET)
+        responses[422] = ('Unprocessable Entity',
+                          'The request was well-formed but was'
+                          ' unable to be followed due to semantic errors')
+
         self.status_code = status or self.default_status
         error_message = "%d: %s" % (self.status_code,
                                     responses[self.status_code][0])
```

Comment:
While this is getting update can we make this a bit more resilient in the case of errors we don't know about get raised?